src/controllers/zcash.js

```diff
@@ -49,22 +49,37 @@
           createChart('#ZcashWalletChart', {
```

Comment:
Looks like you have merge conflicts in this file, I would just revert the changes here. The zcash metrics stuff is all done on the master branch